packages/core/src/Modules/SmartTraderIFrame/SmartTraderIFrame.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const SmartTraderIFrame = () => (
+    <iframe
+        id='localstorage-sync'
+        src='https://aaron-binary3.binary.sx/br_ls-sync/localstorage-sync.html'
```

Comment:
Please add a TODO with a tag to change this origin.

Reply:
No longer applicable.